app/boards/shields/sofle_rgb_keyhive/boards/nice_nano_v2.overlay

```diff
@@ -0,0 +1,38 @@
+#include <dt-bindings/led/led.h>
+
+/*
+ * Copyright (c) 2022 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+&spi1 {
+  compatible = "nordic,nrf-spim";
+  status = "okay";
+  mosi-pin = <6>;
+  // Unused pins, needed for SPI definition, but not used by the ws2812 driver itself.
+  sck-pin = <5>;
+  miso-pin = <7>;
+
+  led_strip: ws2812@0 {
+    compatible = "worldsemi,ws2812-spi";
+    label = "WS2812";
+
+    /* SPI */
+    reg = <0>; /* ignored, but necessary for SPI bindings */
+    spi-max-frequency = <4000000>;
+
+    /* WS2812 */
+    chain-length = <16>; /* number of LEDs */
```

Comment:
Is this correct?
While testing this, I noticed when I enable the lighting all of the bottom lights work, but I believe it is enabling 16 on each half. 
There are 8 under-glow and 8 per-key LEDs that light up per side. 
There are 29 per-key LEDs and 8 underglow.

Apologies if this is the wrong place to post. Didn't see an "Issues" tab on this branch.

Reply:
Yeah, ok. After further testing I am going to have to agree with Nicell. LEDs aren't ready yet.

Reply:
@researcherx00 I've got some hardware issues with my LEDs that I can't fix right now (broken traces). Could you attempt to modify this to 37 (29 + 8) or 8 and confirm that you can control all of the LEDs on each side? Unfortunately, the definition of how "underglow LEDs" work in Zephyr doesn't match how these were implemented on the board.

Reply:
I will try to test it, sure. Will report back when I can.

Reply:
Yep, 37 did the trick. 
I will say the left side is flickery, especially with fade in/fade out animations (and even more specifically when the color is red). Most likely me and shitty soldering skills 🙂 though unless someone else could replicate.
I initially said "Maybe it isn't ready yet" because of the flicker but then realized it could be me. 

Photo of all lights on:
![image](https://user-images.githubusercontent.com/81391847/167945391-e62aac67-43e1-467f-bc41-b4f24b3b9e2d.png)

Reply:
Do we need to update this number to 37 from 16?